flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1241,7 +1241,6 @@ private void failSessionDuringDeployment(YarnClient yarnClient, YarnClientApplic
 			// call (we don't know if the application has been deployed when the error occured).
 			LOG.debug("Error while killing YARN application", e);
 		}
-		yarnClient.stop();
```

Comment:
Why did we remove this? Seems like an unrelated change compared to having a new client in the DeploymentFailureHook